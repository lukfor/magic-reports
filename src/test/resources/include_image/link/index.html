{{image("https://test.com/image.png")}}