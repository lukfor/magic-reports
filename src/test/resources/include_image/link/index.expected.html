<img src="https://test.com/image.png">